packages/react-tinacms-editor/src/types.ts

```diff
@@ -50,3 +51,15 @@ export type NodeViews = {
 export interface Command {
   (state: EditorState, ...options: any[]): void
 }
+
+export interface Plugin {
+  name: string
+  WysiwygMenu: ElementType
```

Comment:
Could we switch this to `MenuItem`? The `Wsyiwyg` part feels redundant.

```suggestion
  MenuItem: ElementType
```

Reply:
actually, the RFC says `menu`

Reply:
I was thinking of possible support for different menus like `WysiwygMenu` for wysiwyg mode, `MarkdownMenu` for raw mode.

But I think this will be ok:

**MenuItem - wysiwyg mode
MarkdownMenuItem - markdown mode**

Markdown menu support is future think.

Reply:
Right. An alternative could be for the `MenuItem` to be passed the `mode` and make the decision from there